client/my-sites/sidebar/sidebar.jsx

```diff
@@ -447,7 +447,7 @@ export class MySitesSidebar extends Component {
 				link={ siteSettingsLink }
 				onNavigate={ this.onNavigate }
 				icon="cog"
-				preloadSectionName="settings"
+				preloadSectionName="settings-general"
```

Comment:
I don't think we need to preload `settings`, it'll load anyway every time when we load any of the settings sections anyway. Could you please describe the regression with more details?